### DISCUSSION TITLE
The icons of web pages are not displayed

### DISCUSSION THREAD

**Discussion by update-freak** [dani-garcia/vaultwarden#1317] [General]
### No Icons
The icons of web pages are not displayed

### Your environment
Server Installed: 1.18.0
Web Installed: 2.17.1
Install method: Synology -> Docker -> bitwardenrs/server -> start Image
Clients used: Android App, Chrome extension
Reverse proxy: used on Synology NAS
Admin page: DNS (github.com) (Error): Could not resolve domain name.

On Synology NAS also Pi-Hole is installed with the following lists:
https://v.firebog.net/hosts/lists.php?type=tick
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/Corona-Blocklist
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/Fake-Science
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/MS-Office-Telemetry
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/Phishing-Angriffe
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/Streaming
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/Win10Telemetry
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/crypto
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/easylist
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/gambling
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/malware
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/notserious
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/samsung
https://raw.githubusercontent.com/RPiList/specials/master/Blocklisten/spam.mails

### Expected behaviour
The icons of web pages are displayed
[log.txt](https://github.com/dani-garcia/bitwarden_rs/files/5824825/log.txt)

**Comment by BlackDex** [comment]
I see that it does work, because an icon for adobe was downloaded.
The only thing which isn't working are the non-global-ip's.

You can change a setting which disables this by default:
https://github.com/dani-garcia/bitwarden_rs/blob/37cc0c34cf5732cbe7b0d4aea622acdd615e3ef6/.env.template#L112

**Comment by update-freak** [comment]
Strange, I don't see the icon in Adobe. In the corresponding folder, where the icons should be, there are only MISS files.

**Comment by BlackDex** [comment]
To get a bit more detailed information you could set the log level to debug. And remove all the .miss files and try again.

That should give a bit more information.

**Comment by update-freak** [comment]
How to set the log level to debug? Is this possible in the admin panel?

**Comment by BlackDex** [comment]
Not sure how this works on a Synology NAS, but you probably have configured some settings somewhere for this container.
It is the following setting: https://github.com/dani-garcia/bitwarden_rs/blob/37cc0c34cf5732cbe7b0d4aea622acdd615e3ef6/.env.template#L79

**Comment by update-freak** [comment]
here is the log-file:
[log20210118.txt](https://github.com/dani-garcia/bitwarden_rs/files/5831649/log20210118.txt)
hope that helps

**Comment by BlackDex** [comment]
Either DNS Resolving isn't working, or your container does not have a connection to the internet.
That is all i can figure out from the logs.

**Comment by update-freak** [comment]
Tried it out. So "apt-get update" worked.
Does this mean that the name resolution is the problem?
